gonanoid.go

```diff
@@ -58,7 +66,7 @@ func Generate(alphabet string, size int) (string, error) {
 		}
 		for i := 0; i < step; i++ {
 			currByte := bytes[i] & byte(mask)
-			if currByte < byte(len(alphabet)) {
+			if int(currByte) < len(alphabet) {
```

Comment:
If we do the check (`len(alphabet) <= 256`) as suggested a few lines earlier there's no need to change this line of code.